Specifications/Language/3_Expressions/ConditionalLoops.md

```diff
@@ -36,21 +34,21 @@ until condition;
 
 where `condition` is an arbitrary expression of type `Bool`.
 
-The `repeat` statement runs a block of statements before evaluating a condition. If the condition evaluates to true, the loop exits. If the condition evaluates to false, an additional block of statements defined as part of an optional `fixup` block, if present, is run prior to entering the next loop iteration. 
+The `repeat` loop runs a block of statements before evaluating a condition. If the condition evaluates to true, the loop exits. If the condition evaluates to false, an additional block of statements defined as part of an optional `fixup` block, if present, is run prior to entering the next loop iteration.
 
-The compiler treats all parts of the `repeat` statement (both blocks and the condition) as a single scope for each repetition; symbols that are defined within the `repeat` block are visible both to the condition and within the `fixup` block. As for other loops, symbols go out of scope after each iteration, such that symbols defined in the `fixup` block are not visible in the `repeat` block.
+The compiler treats all parts of the `repeat` loop (both blocks and the condition) as a single scope for each repetition; symbols that are defined within the `repeat` block are visible both to the condition and within the `fixup` block. As for other loops, symbols go out of scope after each iteration, such that symbols defined in the `fixup` block are not visible in the `repeat` block.
```

Comment:
This one actually isn't true anymore... it was a hard feature to support and was unfortunately bugprone, so now each block in the repeat-expression is treated as it's own scope, similar to if-expressions (see [this code sample](https://microsoft.github.io/qsharp/?code=H4sIAAAAAAAAE12OvWrDQBCE%2B3uKATV2EyVtQiBFgiuDlaQzLtbS2l64P%2B72iI3RuwedgxGZ8pudmW3bFg2%2ByEXLpm1brFhV%2FBFZKSkPE6u8wTvnPklUCb6S75NkSAbBiRdHFl2DmMIxkYOeSNGTx55RMg%2FQcKvET5I60DXow8APxpPjHKlnrC9dIa%2FFbf5arsYAwNuH13TZBPG6WFYSIieaPsGaxC%2BWeMYn52J1u8O1XkxKHJl0Bia5orS3jDNe8fhyt0YUr2JxIJv5Tg9yLvFfQWat4adZeDapJXlsdzdzNKP5BSRayv1jAQAA)). The expected workaround is to declare any variables that need to be visible across the scopes outside of the repeat-expression.

Reply:
Okay, I think the idea that each of the blocks create their own scope is the expected behavior of blocks in general, so we should be fine to just drop the paragraph rather than try to fix it.